x-pack/legacy/plugins/monitoring/public/components/status_icon/index.js

```diff
@@ -5,22 +5,27 @@
  */
 
 import React from 'react';
+import {
+  EuiIcon,
+} from '@elastic/eui';
 
 export function StatusIcon({ type, label }) {
   const typeToIconMap = {
-    [StatusIcon.TYPES.RED]: 'health-red.svg',
-    [StatusIcon.TYPES.YELLOW]: 'health-yellow.svg',
-    [StatusIcon.TYPES.GREEN]: 'health-green.svg',
-    [StatusIcon.TYPES.GRAY]: 'health-gray.svg',
+    [StatusIcon.TYPES.RED]: 'danger',
+    [StatusIcon.TYPES.YELLOW]: 'warning',
+    [StatusIcon.TYPES.GREEN]: 'success',
+    [StatusIcon.TYPES.GRAY]: 'subdued',
   };
   const icon = typeToIconMap[type];
 
   return (
     <span className="kuiStatusText">
```

Comment:
Since you're in here and changing this file, can you remove the need for `kuiStatusText`? It's part fo the old lib, and we'd love to remove it.